.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: "Unit test"
+on:
+  - "push"
```

Comment:
Quotes unnecessary, but fine.

Reply:
I've got into the habit of always quoting strings in YAML :)